testing/tests/DevExpress.ui.widgets.editors/datebox.tests.js

```diff
@@ -3677,6 +3677,26 @@ QUnit.test("datebox should be invalid after out of range value was setted", (ass
     assert.ok(dateBox.option("isValid"), "widget is valid");
 });
 
+QUnit.test("datebox should change validation state if value was changed by keyboard", (assert) => {
+    const $dateBox = $("#dateBox").dxDateBox({
+        type: "date",
+        value: null
+    }).dxValidator({
+        validationRules: [{
+            type: "required",
+            message: "Date of birth is required"
+        }]
+    });
+    const dateBox = $dateBox.dxDateBox("instance");
+    const keyboard = keyboardMock($dateBox.find("input"));
```

Comment:
```suggestion
    const keyboard = keyboardMock($dateBox.find(`.${TEXTEDITOR_INPUT_CLASS }`));
```
Editor could have more than one input element